Library/Homebrew/dev-cmd/man.rb

```diff
@@ -192,7 +192,8 @@ def cmd_comment_manpage_lines(cmd_path)
                  .each do |line|
       line = line.slice(4..-1)
       next unless line
-      lines << line.gsub(/^ +(-+[a-z-]+) */, "* `\\1`:\n  ")
+      lines << line.gsub(/^ +(-+[a-z-]+), (-+[a-z-]+) +/, "* `\\1`, `\\2`:\n  ")
```

Comment:
What's this change for?

Reply:
This function is only used in parsing the help string comment in shell commands. The current regexp grabs the first word as the option and the rest of the line is description. Without this change, a line like this

    #:    -v, --verbose                           Be verbose

would end up formatted like this

    * `-v`:
      --verbose                           Be verbose

instead of what we want

    * `-v`, `--verbose`:
      Be verbose

So this change is necessary if we want to allow the help string to list both short and long options, makes the raw help as well as the markdown consistent with ruby `OptionParser`-based commands.

Reply:
Gotcha, thanks 👍